pkg/blobplugin/Dockerfile

```diff
@@ -28,8 +28,10 @@ RUN chmod +x /blobfuse-proxy/init.sh && \
  chmod +x /blobfuse-proxy/blobfuse-proxy.service && \
  chmod +x /blobfuse-proxy/blobfuse-proxy
 
-RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates uuid-dev util-linux mount udev wget e2fsprogs nfs-common netbase
-
+RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates uuid-dev util-linux mount udev wget e2fsprogs nfs-common netbase procps conntrack iptables bind9-host iproute2 bash netcat sysvinit-utils
+# install aznfs
+RUN wget -O aznfs.tar.gz https://github.com/Azure/AZNFS-mount/releases/download/0.1.116/aznfs-0.1.116.tar.gz
+RUN tar xvzf aznfs.tar.gz -C / && rm aznfs.tar.gz
```

Comment:
can you verify whether this change works on Mariner 2.0 and arm64? this PR should not break on those clusters at least, thanks.